### ISSUE TITLE
bls_api returns df as type function

### ISSUE THREAD

**Issue by smitty1788** [keberwein/blscrapeR#16]
While running the bls_api, it is converting the dataframe it should create as a function. The function looks like below

```
function (x, df1, df2, ncp, log = FALSE) 
{
  if (missing(ncp)) 
    .Call(C_df, x, df1, df2, log)
  else .Call(C_dnf, x, df1, df2, ncp, log)
}
```
Here is my full code I am using
```
library(tidyverse)
library(blscrapeR)

msa <- read.delim("https://raw.githubusercontent.com/smitty1788/Personal-Website/master/dl/MSA_Codes.txt",
                header = FALSE) %>% 
  rename(area_code = V2, area_text = V3) %>% 
  select(area_code, area_text) %>% 
  mutate(CBSA_ID = substr(area_code, 5, 9))

MSA_LAUS <- tibble(date = character(),
                   labor_force = numeric(),
                   employment = numeric(),
                   unemployment = numeric(),
                   unemployment_rate = numeric(),
                   CBSA_ID = character(),
                   area_text = character())

for (i in 1:395) {
  ac <- as.character(msa$CBSA_ID[i])
  at <- as.character(msa$area_text[i])
  
  labor_force <- as.character(paste0("LAU", msa$area_code[i], "06"))
  employment <- as.character(paste0("LAU", msa$area_code[i], "05"))
  unemployment <- as.character(paste0("LAU", msa$area_code[i], "04"))
  unemployment_rate <- as.character(paste0("LAU", msa$area_code[i], "03"))
  
  x <- as.vector(c(labor_force, employment, unemployment, unemployment_rate))
  series_id <- tibble(
    name = c("labor_force", "employment", "unemployment", "unemployment_rate"),
    id = c(labor_force, employment, unemployment, unemployment_rate)
  )
  
  labor <- bls_api(x, startyear = '2002', endyear = '2010')
  laus <- labor %>% 
    mutate(seriesID = as.character(seriesID)) %>% 
    merge(series_id, by.x = 'seriesID', by.y = 'id', all.x = TRUE) %>%
    filter(!is.na(name)) %>% 
    select(year, periodName, name, value) %>% 
    spread(name, value = value) %>% 
    mutate(date = as.Date(paste0(as.character(year), " ", as.character(periodName), " ", "01"), '%Y %B %d'),
           CBSA_ID = ac,
           area_text = at) %>% 
    select(date, labor_force, employment, unemployment, unemployment_rate, CBSA_ID, area_text)
  
  MSA_LAUS <- rbind(MSA_LAUS, laus)
}
```

The error returns as 
```
Error in UseMethod("mutate_") : 
  no applicable method for 'mutate_' applied to an object of class "function"
```

**Comment by keberwein** [comment]
See my comment in https://github.com/keberwein/blscrapeR/issues/13#issuecomment-338000973

**Comment by keberwein** [comment]
@smitty1788 I was able to reproduce your error this morning. I believe we are exceeding our daily request limits. When I went into the function (using my API key), I received the following message on my JSON request `[1] "Request could not be serviced, as the daily threshold for total number of requests allocated to the user has been reached."`

This made sense to me since I ran the full loop of 395 requests less than 24 hours ago. Just to double-check, I went and signed up for a new API key with a different email address and ran the full loop again, and it worked! 

I don't think the problem is your code or mine, I think we're just hitting our request limit. The BLS documentation specifies "500 requests daily," but they don't define what they consider a "day." Looks to me like it's 24-hours.

Again, I've attached my .RData file of the completed run. 

[msa_laus_2017-10-20.RData.zip](https://github.com/keberwein/blscrapeR/files/1402285/msa_laus_2017-10-20.RData.zip)
